src/si/frequency_drift.rs

```diff
@@ -0,0 +1,61 @@
+//! Frequency drift (base unit hertz per second, s⁻²).
+//!
+//! Typical application: Frequency slope in FMCW radars.
+
+quantity! {
+    /// Frequency drift (base unit hertz per second, s⁻²).
+    quantity: FrequencyDrift; "frequency drift";
+    /// Dimension of frequency drift, T⁻² (base unit hertz per second, s⁻²).
+    dimension: ISQ<
+        Z0,     // length
+        Z0,     // mass
+        N2,     // time
+        Z0,     // electric current
+        Z0,     // thermodynamic temperature
+        Z0,     // amount of substance
+        Z0>;    // luminous intensity
+    units {
```

Comment:
Will you sort the units from largest prefix to smallest. Terahertz/s got joined into the gigahertz line. Do the same for the tests at the bottom of the file.